packages/ketcher-react/src/script/editor/shared/closest.js

```diff
@@ -454,22 +454,29 @@ function findClosestItem(restruct, pos, maps, skip, scale) {
   // eslint-disable-line max-params
```

Comment:
I found the same function in in Core - packages/ketcher-core/src/application/editor/shared/closest.js:415
Do we need to update this one also?